src/core/plugin/luapi_application.h

```diff
@@ -1673,14 +1673,16 @@ static int applib_export(lua_State* L) {
 
     lua_getfield(L, 1, "outputFile");
     lua_getfield(L, 1, "range");
+    lua_getfield(L, 1, "layerRange");
```

Comment:
Could you please replace `["range"] = "1-2"` by `["layerRange"] = "1-2"` in Example 3 above? Otherwise those writing a plugin might miss that there is an option to export only selected layers.